packages/date/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
+## 2.1.1 (Unreleased)
+
+### Deprecations
+
+- Remove `moment` from the public API for the date module.
```

Comment:
Seems like a pretty breaking change, might be worth at least a minor version bump rather than the bug fix bump.

Reply:
I guess this is just deprecating it, but I'd think that's a bigger change. Whatever, either way is fine 😄 🤷‍♂️

Reply:
The deprecation should be a minor release, not patch though.

>Minor version [...] must be incremented if any public API functionality is marked as deprecated.

https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs